dts/bindings/ethernet/atmel,sam-gmac.yaml

```diff
@@ -15,3 +16,32 @@ properties:
       type: int
       required: true
       description: Number of hardware TX and RX queues
+
+    local-mac-address:
+      type: uint8-array
+      description:
+        Specifies the MAC address that was assigned to the network device
```

Comment:
`local-mac-address` is already defined in `ethernet.yaml`, so no need to re-define it here; though the property description there could probably use some improvement.

Reply:
indeed, fixed!